src/ws/test-auth.c

```diff
@@ -749,10 +749,22 @@ static const ErrorFixture fixture_non_ascii = {
 
 static const ErrorFixture fixture_tls_cert = {
   .error_code = COCKPIT_ERROR_AUTHENTICATION_FAILED,
-  .error_message = "Authentication required",
+  .error_message = "Unsupported Authorization type",
   .header = "tls-cert e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
 };
 
+static const SuccessFixture fixture_bearer = {
+  .warning = NULL,
+  .data = NULL,
+  .header = "Bearer correcttoken"
+};
+
+static const ErrorFixture fixture_invalid_bearer = {
+  .error_code = COCKPIT_ERROR_AUTHENTICATION_FAILED,
+  .error_message = "Unsupported Authorization type",
+  .header = "Bearer ocrrecttoken"
```

Comment:
Can you please use `wrongtoken` here, to better convey that this is not a typo, but a deliberately wrong password for testing?